src/components/KeyboardShortcuts/index.tsx

```diff
@@ -16,10 +16,20 @@ import {
   actions as versionsActions,
   goToRelativeDiff,
 } from '../../reducers/versions';
+import { actions as fullscreenGridActions } from '../../reducers/fullscreenGrid';
 import styles from './styles.module.scss';
 import { gettext } from '../../utils';
 
-export const supportedKeys = ['k', 'j', 'e', 'o', 'c', 'n', 'p'];
+export const supportedKeys: { [key: string]: string | null } = {
+  k: gettext('Up file'),
+  j: gettext('Down file'),
+  o: gettext('Open all folders'),
+  e: null, // same as 'o'
+  c: gettext('Close all folders'),
+  n: gettext('Next change'),
+  p: gettext('Previous change'),
+  h: gettext('Toggle side panels'),
```

Comment:
As a follow-up to my other suggestion, this would need to change

Reply:
Ok, I changed for "Hide file tree"

Reply:
mmh, I wonder if we don't want to change to something like "Hide left side panel", because if one has a different accordion item expanded, it's not really the "file tree" that will be collapsed 🤔